packages/logs/lib/client.ts

```diff
@@ -67,7 +67,14 @@ export class LogContextStateless {
             type: 'log',
             level: 'error',
             message,
-            error: err ? { name: err.name, message: err.message } : null,
+            error: err
+                ? {
+                      name: error instanceof Error ? error.constructor.name : err.name,
+                      message: err.message,
+                      type: 'type' in err ? (err.type as string) : null,
+                      payload: 'payload' in err ? err.payload : null
```

Comment:
can we use `??`:
```
      type: err.type as string ?? null,
      payload: err.payload ?? null
```

Reply:
type and payload do not exists in regular Error put I could put Error | NangoError in the inital type to help

Reply:
edit: nevermind it returns an object so its the only "typesafe" way. Note that I could